README.md

```diff
@@ -101,6 +101,7 @@ services:
       - PGID=1000
       - APP_URL=
       - DB_HOST=bookstack_db
+      - DB_PORT=3306
```

Comment:
changes must be made in readme_vars

Reply:
Thanks for review, I will revert that changes

Reply:
I've reverted the README changes and updated to readme-vars.yml instead